extension.ts

```diff
@@ -127,6 +127,11 @@ export async function activate(context: vscode.ExtensionContext) {
     }, 0);
   });
 
+  vscode.workspace.onDidCloseTextDocument((event) => {
+    // Remove modehandler for closed document
+    delete modeHandlerToEditorIdentity[event.fileName + vscode.window.activeTextEditor.viewColumn];
```

Comment:
I'm not very familiar with what this does. When is an editorIdentity created with a key `event.fileName + vscode.window.activeTextEditor.viewColumn`?